lib/perl/Genome/Model/Command/Define.pm

```diff
@@ -24,6 +24,7 @@ sub _build_sub_command {
 
     my $model_subclass_name = $class_name;
     $model_subclass_name =~ s/::Command::Define::/::/;
+    return if eval{ $model_subclass_name->do_not_create_define_command; };
```

Comment:
This should use Try::Tiny to protect $@.

Alternatively, you could define ```do_not_create_define_command()``` in Genome::Model to return false and do away with the eval/try.